hasher-matcher-actioner/webapp/src/Api.jsx

```diff
@@ -139,24 +139,48 @@ export async function submitContent(
   submissionType,
   contentId,
   contentType,
-  contentRef,
-  metadata,
+  content,
+  additionalFields,
 ) {
   return apiPost('/submit/', {
     submission_type: submissionType,
     content_id: contentId,
     content_type: contentType,
-    content_ref: contentRef,
-    metadata,
+    content_bytes_url_or_file_type: content,
+    additional_fields: additionalFields,
   });
 }
 
-export async function submitContentUpload(
+export async function submitContentPostURLUpload(
   submissionType,
   contentId,
   contentType,
-  contentRef,
-  metadata,
+  content,
+  additionalFields,
+) {
+  const postURL = await apiPost('/submit/', {
```

Comment:
This name threw me for a sec... it's the response object from the apiPost call... What about `response` or `submitContentResponse` (whatever matches the python dataclass that's returned)? Nit-ish. Non-blocking.

Reply:
Thanks! will update name and add comment.